src/templates/kit/screens/DotVotingScreen.js

```diff
@@ -0,0 +1,342 @@
+import React, { useCallback, useReducer, useState, useRef } from 'react'
+import PropTypes from 'prop-types'
+import { GU, Help, Info } from '@aragon/ui'
+import {
+  Duration,
+  Header,
+  KnownAppBadge,
+  Navigation,
+  PercentageField,
+  ScreenPropsType,
+} from '..'
+
+const MINUTE_IN_SECONDS = 60
+const HOUR_IN_SECONDS = MINUTE_IN_SECONDS * 60
+const DAY_IN_SECONDS = HOUR_IN_SECONDS * 24
+
+const DEFAULT_SUPPORT = 0
```

Comment:
Is this the appropriate default for support?

Reply:
Yeah, this is appropriate. Thanks for asking. Dot Voting is a bit different as it's a mult-option vote -- basically you can end up with a final distribution as follows:

option 1 - 50%
option 2 - 20%
option 3 - 15%
option 4 - 10%
option 5 - 5%

Right now -- all options will be valid when support is set as 0. If you set support to something like "6%" then the 5% option wouldn't get a portion of the allocation. A bit different mechanics than binary voting. Make sense?

Reply:
Yes, thank you for the clarification.

Reply:
I think we should make sure to clarify about this in our documentation @stellarmagnet @jayalaves I was wondering the same myself in the past, it seems it will be a recurrent question